modules/bmp/image_loader_bmp.cpp

```diff
@@ -32,6 +32,15 @@
 
 #include "core/io/file_access_memory.h"
 
+static uint8_t get_mask_width(uint16_t mask) {
+	// Returns number of ones in the binary value of the parameter: mask.
+	// Uses a Simple pop_count.
+	uint8_t c = 0u;
+	for (; mask != 0u; mask &= mask - 1u)
+		c++;
```

Comment:
```suggestion
	for (; mask != 0u; mask &= mask - 1u) {
		c++;
	}
```

Our style is to always use braces around ``if`` statements

Reply:
I added braces. Thank you for your suggestion.